morebits.js

```diff
@@ -371,7 +371,7 @@ Morebits.quickForm.prototype.append = function QuickFormAppend(data) {
  * There is some difference on how types handle the `label` attribute:
  * - `div`, `select`, `field`, `checkbox`/`radio`, `input`, `textarea`, `header`, and `dyninput` can accept an array of items,
  * and the label item(s) can be `Element`s.
- * - `option`, `optgroup`, `_dyninput_element`, `submit`, and `button` accept only a single string.
+ * - `option`, `optgroup`, `_dyninput_cell`, `submit`, and `button` accept only a single string.
```

Comment:
For my understanding, what does _dyninput_cell do and why did its name change?

Reply:
`_dyninput_element` was an internal type used by `dyninput`, and is now renamed `_dyninput_cell` because there's now `_dyninput_row`.